docs/best-practices/laying-out-apps.md

```diff
@@ -161,6 +161,14 @@ Stack(
 
 ```
 
+:::caution
+
+**When dealing with `CausalGraphViewer`, `Table`, `Plotly` and `Scrollable Stacks`:**
```

Comment:
If we put scrollable stacks in capital letters and within `code brackets` it seems like it is its own component which could send someone looking for it. could we do scrollable `Stack`s?

Reply:
fair point, fixed